lib/benchmark.rb

```diff
@@ -314,18 +314,33 @@ def measure(label = "") # :yield:
 
   #
   # Returns the elapsed real time used to execute the given block.
-  # The unit of time is seconds.
+  # The unit of time is seconds by default, or can be specified with the unit parameter.
+  # See documentation for `Process.clock_gettime`` for supported units.
   #
   #       Benchmark.realtime { "a" * 1_000_000_000 }
   #       #=> 0.5098029999935534
   #
-  def realtime # :yield:
-    r0 = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+  #       Benchmark.realtime(:float_millisecond) { "a" * 1_000_000_000 }
+  #       #=> 509.8029999935534
+  #
+  def realtime(unit = :float_second) # :yield:
+    r0 = Process.clock_gettime(Process::CLOCK_MONOTONIC, unit)
```

Comment:
One concern here is there might be a non-trivial overhead to pass a variable unit to `Process.clock_gettime`, which happens with this change. Ruby implementations might optimize it better if the unit is constant.

So I think it would be safer to:
* Leave `realtime` untouched
* Add `ms` and inline the logic (it's so simple anyway), and it also avoids an extra method call + block call for `ms`.